integration_tests/integration_test.go

```diff
@@ -24,7 +24,16 @@ func init() {
 	}
 }
 
-func TestSpinnakerBase(t *testing.T) {
+func TestIntegrationTests(t *testing.T) {
+	t.Run("Operator=1", SpinnakerBase)
```

Comment:
We are now executing the tests sequentially, since we are using a local cluster it is good to not overload it.

Reply:
How much does that impact overall integration test times?

Reply:
It does impact substantially, the normal execution timeline on the current execution on the spinnaker pipeline is roughly 15 - 20 minutes.
Not it could take up to 40 minutes.

Reply:
Okay, that does sound like quite a bit. Let's keep an eye on PR times and see if we need to reconsider this change.